src/core_modules/capture-core/components/WorkingLists/TeiWorkingLists/epics/teiViewEpics/helpers/getTeiListData/getTeiListData.js

```diff
@@ -37,17 +37,10 @@ const getMainApiFilterQueryArgs = (filters?: RawFilterQueryArgs = {}, filtersOnl
             };
         }, {});
 
-const getApiOrderById = (sortById: string, columnsMetaForDataFetching: TeiColumnsMetaForDataFetching) => {
-    const { id, apiName } = columnsMetaForDataFetching.get(sortById) || {};
-    return apiName || id;
-};
-
 const getApiOrderByQueryArgument = (sortById: string, sortByDirection: string, columnsMetaForDataFetching: TeiColumnsMetaForDataFetching) => {
-    const apiId = getApiOrderById(sortById, columnsMetaForDataFetching);
-    return `${apiId}:${sortByDirection}`;
+    const { id } = columnsMetaForDataFetching.get(sortById) || {};
+    return id ? `${id}:${sortByDirection}` : 'createdAt:desc';
```

Comment:
Do you think it makes sense to change the logic so that the only time that we validate is when we load the view? I.e., we assume that sortById and sortByDirection is valid here, but when we actually load a view we check that the sort by column is actually one of the columns.